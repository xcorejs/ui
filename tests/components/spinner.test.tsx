import 'jest-styled-components';

import { Spinner } from '@xcorejs/ui';
import React from 'react';
import renderer from 'react-test-renderer';

test('Spinner component', () => {
  const component = renderer.create(
    <>
      <Spinner />
    </>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
