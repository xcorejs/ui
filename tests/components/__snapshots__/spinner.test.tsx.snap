// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component 1`] = `
.c0 {
  display: inline-block;
  border-color: currentColor;
  border-style: solid;
  border-radius: full;
  border-width: 2px;
  border-left-color: transparent;
  border-bottom-color: transparent;
  width: 1.6rem;
  height: 1.6rem;
  -webkit-animation: cilQsd 0.5s linear infinite;
  animation: cilQsd 0.5s linear infinite;
}

<div
  className="c0"
  speed="0.5s"
/>
`;
